### PR TITLE
Recipe Registering, Screens, Material names and Sounds

### LINKED ISSUE #34: Mutated biomes don't have names

### LINKED ISSUE #35: Recipe registry classes aren't mapped

### LINKED ISSUE #36: Some block and sound parameters have incorrect names

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Recipe registering classes were pretty easy to map as they were all kinda the same. The material fields were mapped with a simple ctrl + c, which I expect should have been done for other material fields.
*please don't squash merge because it makes collaborating much harder*

Closes #34 
Closes #35 
Closes #36 

Changelog:-
```applescript
 Changes to be committed:
	modified:   mappings/net/minecraft/client/options/GameOptions.mapping
	modified:   mappings/net/minecraft/client/render/WorldRenderer.mapping
	modified:   mappings/net/minecraft/client/sound/AbstractSoundInstance.mapping
	new file:   mappings/net/minecraft/client/sound/GuardianAttackSoundInstance.mapping
	modified:   mappings/net/minecraft/client/sound/MinecartSoundInstance.mapping
	modified:   mappings/net/minecraft/client/sound/SoundCategory.mapping
	modified:   mappings/net/minecraft/client/sound/SoundEntry.mapping
	modified:   mappings/net/minecraft/client/sound/SoundInstance.mapping
	modified:   mappings/net/minecraft/client/sound/SoundManager.mapping
	modified:   mappings/net/minecraft/client/sound/SoundRegistry.mapping
	modified:   mappings/net/minecraft/client/sound/SoundSystem.mapping
	new file:   mappings/net/minecraft/server/world/ServerWorldManager.mapping
	deleted:    mappings/net/minecraft/sound/GuardianAttackSoundInstance.mapping
	modified:   mappings/net/minecraft/util/math/Vec3i.mapping
	new file:   mappings/net/minecraft/class_768.mapping
	new file:   mappings/net/minecraft/client/gui/TwitchErrorScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/AchievementsScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/CommandBlockScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/CustomizedWorldPresetsScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/ResourcePackScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/SnooperScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/SoundsScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/StreamIngestScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/StreamScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/StreamUtilitiesScreen.mapping
	new file:   mappings/net/minecraft/client/gui/screen/ingame/VillagerTradingScreen.mapping
	modified:   mappings/net/minecraft/client/sound/SoundSystem.mapping
	new file:   mappings/net/minecraft/client/texture/ClockSprite.mapping
	new file:   mappings/net/minecraft/client/texture/CompassSprite.mapping
	modified:   mappings/net/minecraft/scoreboard/AbstractTeam.mapping
	modified:   mappings/net/minecraft/util/Util.mapping
	modified:   mappings/net/minecraft/util/math/Box.mapping
	modified:   mappings/net/minecraft/util/snooper/Snoopable.mapping
	modified:   mappings/net/minecraft/block/BedBlock.mapping
	modified:   mappings/net/minecraft/block/BigMushroomBlock.mapping
	modified:   mappings/net/minecraft/block/Block.mapping
	modified:   mappings/net/minecraft/block/Blocks.mapping
	modified:   mappings/net/minecraft/block/CakeBlock.mapping
	modified:   mappings/net/minecraft/block/PressurePlateBlock.mapping
	modified:   mappings/net/minecraft/block/SandstoneBlock.mapping
	modified:   mappings/net/minecraft/block/Material.mapping
	new file:   mappings/net/minecraft/block/PortalMaterial.mapping
	modified:   mappings/net/minecraft/client/render/RenderLayer.mapping
	modified:   mappings/net/minecraft/block/PrismarineBlock.mapping
	modified:   mappings/net/minecraft/block/QuartzBlock.mapping
	modified:   mappings/net/minecraft/block/StoneBrickBlock.mapping
	new file:   mappings/net/minecraft/block/entity/SpawnerBlockEntityBehavior.mapping
	new file:   mappings/net/minecraft/class_1684.mapping
	new file:   mappings/net/minecraft/enchantment/BetterLootEnchantment.mapping
	modified:   mappings/net/minecraft/enchantment/Enchantment.mapping
	modified:   mappings/net/minecraft/enchantment/EnchantmentTarget.mapping
	modified:   mappings/net/minecraft/entity/mob/MobEntity.mapping
	modified:   mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping
	new file:   mappings/net/minecraft/recipe/ArmorRecipes.mapping
	new file:   mappings/net/minecraft/recipe/BlockRecipes.mapping
	new file:   mappings/net/minecraft/recipe/CombatItemRecipes.mapping
	new file:   mappings/net/minecraft/recipe/FoodRecipes.mapping
	new file:   mappings/net/minecraft/recipe/HoeRecipes.mapping
	new file:   mappings/net/minecraft/recipe/NineGoldRecipes.mapping
	modified:   mappings/net/minecraft/recipe/Recipes.mapping
	new file:   mappings/net/minecraft/recipe/ShapelessRecipes.mapping
	new file:   mappings/net/minecraft/recipes/SmeltingRecipes.mapping
	modified:   mappings/net/minecraft/util/JsonSet.mapping
	renamed:    mappings/net/minecraft/class_1684.mapping -> mappings/net/minecraft/util/JsonElementProvider.mapping
	deleted:    mappings/net/minecraft/class_133.mapping
	new file:   mappings/net/minecraft/class_595.mapping
	new file:   mappings/net/minecraft/world/biome/EndBiomePopulator.mapping
	new file:   mappings/net/minecraft/world/biome/MutatedBiome.mapping
	modified:   mappings/net/minecraft/world/biome/SavannaBiome.mapping
	new file:   mappings/net/minecraft/world/gen/feature/FillerBlockFeature.mapping
```

### CODE REVIEW

File: mappings/net/minecraft/block/Blocks.mapping [hunk 1]
```diff
@@ -38,6 +38,7 @@ CLASS net/minecraft/class_163 net/minecraft/block/Blocks
 	FIELD field_660 CACTUS Lnet/minecraft/class_169;
 	FIELD field_661 CLAY Lnet/minecraft/class_160;
 	FIELD field_662 AIR Lnet/minecraft/class_160;
+		COMMENT
```
Comment (shedaniel) [inline mappings/net/minecraft/block/Blocks.mapping:41]: What's happening with these?

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/block/Blocks.mapping:41]: I thought of including Javadoc to constants but then I decided to keep it for later

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/block/Blocks.mapping:41]: It Isn't getting removed for some reason, I'll keep it until I fix it

Comment (shedaniel) [reply mappings/net/minecraft/block/Blocks.mapping:41]: Alrighty

**Pull request merged**